CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.0.1 (19.11.2020)
+
+Small refactoring of `snapshot` command.
+
+Improved error output when project build fails.
+
+Updated [Snapshot \[options\]](README.md#snapshot-options) and
+[What is NO_HASH?](README.md#what-is-no_hash) sections in [README.md](README.md).
+
+Update dependencies.
```

Comment:
If it was a library, it would be important due to breaking changes, peer deps, etc. But Harold is a CLI tool, so I think it isn't useful for CHANGELOG readers.